packages/create-cloudflare/src/common.ts

```diff
@@ -33,17 +33,18 @@ export const validateProjectDirectory = (relativePath: string) => {
 	const existsAlready = existsSync(path);
 	const isEmpty = existsAlready && readdirSync(path).length === 0; // allow existing dirs _if empty_ to ensure c3 is non-destructive
 
-	if (existsAlready && !isEmpty) {
-		crash(
-			`Directory \`${relativePath}\` already exists and is not empty. Please choose a new name.`
-		);
+	if (relativePath && existsAlready && !isEmpty) {
```

Comment:
Do we need to check `relativePath` is truthy? If it is an empty string, `existsAlready` will be `false`

Reply:
We do if we want to have `validateProjectDirectory` work with both the interactive prompt as well as `setupProjectDirectory`. 

The issue is with the way the clack text prompt is handling user input. If you run c3 with current main and hit `enter` when prompted with the default project directory it will fail with `Directory "" already exists and isn't empty`. Clack handles empty string input by replacing it with defaultValue where supplied, but it only does so AFTER running the validator. When the validator doesn't allow empty string, it fails, and clack doesn't get a chance to default it.

Reply:
I think this was fixed by @petebacondarwin in #3496

Reply:
Cool, reverted this bit, but still returning error as string instead of crashing